packages/client/src/index.ts

```diff
@@ -107,6 +107,7 @@ export class RelayClient extends EventEmitter {
     const eventWriteRequest = {
       event: signedEvent,
     };
+    this.eventStream.enqueue({ event: shopEvent });
```

Comment:
we should add a tests in the client to show that we are getting the event from the eventStream that we are writing.